infrastructure/config.sh

```diff
@@ -17,9 +17,9 @@
 # Variables values that will be used in init script to create resources for XC3 infrastructure
 export aws_region="eu-west-1"
 export dynamo_table_name="terraform-lock"
-export bucket_name="terraform-state-xc3"
-export project="example"
-export domain="testing.example.co"
-export owner_email="admin@example.co"
-export creator_email="admin@example.co"
-export namespace="example"
+export bucket_name="terraform-state-xc3-test"
```

Comment:
change test from all fields

Reply:
Added in e61ade95cc3e8225564f1186c07882924e127a83